### PR TITLE
explicitly disable leader election if it is not enabled

### PR DESCRIPTION
**Pull request by MHBauer**
I think since most people will use the chart, this is better than doing it in the code.

WDYT?

### CODE REVIEW

**Comment by MHBauer** [comment]
A symlink loop? Travis are you drunk?

**Comment by MHBauer** [comment]
@duglin looks like something weird is going on in the href checker

**Comment by duglin** [comment]
hold on -fixing it

**Comment by duglin** [comment]
hold on -fixing it

**Comment by duglin** [comment]
can we kick the build?

**Comment by MHBauer** [comment]
Backstory here is that we were seeing permissions failures even with the RBAC PR #936 being used to provision with helm.

File: charts/catalog/templates/controller-manager-deployment.yaml [hunk 1]
```diff
@@ -41,6 +41,8 @@ spec:
         - "8080"
         {{ if .Values.controllerManager.leaderElectionNamespace.activated -}}
```
Comment (pmorie) [inline charts/catalog/templates/controller-manager-deployment.yaml]: Can we change this in the chart? `leaderElectionNamespace` -> `leaderElection`

Comment (luxas) [inline charts/catalog/templates/controller-manager-deployment.yaml]: +1

**Comment by luxas** [comment]
ping @MHBauer

File: charts/catalog/templates/controller-manager-deployment.yaml (hunk 1 above)
Comment (MHBauer) [inline charts/catalog/templates/controller-manager-deployment.yaml]: might as well while I'm in here.

**Comment by MHBauer** [comment]
rebased and nits addressed.

**Pull request merged**